Tests/Util/TestAccountFactory.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  Copyright (c) 2020-2023 MobileCoin. All rights reserved.
+//
+@testable import MobileCoin
+import XCTest
+
+#if swift(>=5.5)
+
+@available(iOS 15.0, *)
+
+public struct TestAccountFactory {
+
+    enum TestAccountFactoryError: Error {
+        case unknownError(_ description: String)
+    }
+
+    let fogReportUrl: String
+    let fogReportId: String = ""
+    let fogAuthoritySpki: Data
+    let accountIndex: UInt32 = 0
+
+    public struct TestAccountConfig {
+        let name: String
+        let txData: [TokenId: [UInt64]]
+    }
+
+    public struct TestAccount {
```

Comment:
we could store the config object and make `name` a computed variable then we have one source of truth for it.

Reply:
done